bin/common.ml

```diff
@@ -1142,9 +1142,9 @@ let context_arg ~doc =
     & opt Arg.context_name Dune_engine.Context_name.default
     & info [ "context" ] ~docv:"CONTEXT" ~doc)
 
-let default () =
+let for_complete () =
   let info = Cmd.info "term-default" in
   let cmd = Cmd.v info term in
-  match Cmd.eval_value ~argv:[| Sys.argv.(0) |] cmd with
+  match Cmd.eval_value ~argv:[| Sys.argv.(0); "--display=quiet" |] cmd with
```

Comment:
You can do this internally in Build_cmd.complete now.

Reply:
What do you mean? I can change `display` but it's still not possible to get a `Common.t` value without parsing a CLI.